include/code_variables.h

```diff
@@ -1882,12 +1880,11 @@ extern MtxF MtxF_clear;
 // extern UNK_TYPE B_80142BBE_jp;
 // extern UNK_TYPE B_80142BC0_jp;
 // extern UNK_TYPE B_80142BE0_jp;
-// extern UNK_TYPE B_80142C00_jp;
+// extern UNK_TYPE fake_table;
```

Comment:
What? lmao

Reply:
This is just me doing Find and Replace and not removing the entry here lol